pkg/instructor/options.go

```diff
@@ -24,13 +26,20 @@ func WithMaxRetries(maxRetries int) Options {
 	return Options{MaxRetries: toPtr(maxRetries)}
 }
 
+func WithValidator(withValidator bool) Options {
```

Comment:
we can either use this name or `WithValidation`

Reply:
changed it to WithValidation. sounds better!